library/kotlin/src/io/envoyproxy/envoymobile/HTTPClient.kt

```diff
@@ -22,8 +21,7 @@ interface HTTPClient {
    * @param responseHandler the callback for receiving stream events.
    * @return CancelableStream, a cancelable request.
    */
-  fun send(request: Request, body: ByteBuffer?, trailers: Map<String, List<String>>,
-                responseHandler: ResponseHandler): CancelableStream
+  fun send(request: Request, body: ByteBuffer?, trailers: Map<String, List<String>>, responseHandler: ResponseHandler): CancelableStream
```

Comment:
Matches `EnvoyClient.kt` - at some point we should probably add a column length linter for kotlin though